index.html

```diff
@@ -207,6 +207,10 @@ <h3>Pinning buffers</h3>
               <p>
               Helpful trigger to automatically repin a buffer (in this instance, <var>irc.freenode.#weechat</var>): <pre><code>/trigger add autopin signal "buffer_opened" "${buffer[${tg_signal_data}].full_name} =~ <var>irc.freenode.#weechat</var>" "" "/command -buffer ${buffer[${tg_signal_data}].full_name} * /buffer set localvar_set_pinned true"</code></pre>
               </p>
+              <h3>Setting path</h3>
+              <p>
+                The path is by default 'weechat'. In case a proxy is used the path can be changed by entering it in the host field. For example <code>your.domain.com:8000/otherpath</code>.
```

Comment:
This should be made clearer, an example isn't a good explanation. It should:
- have a link to the reverse proxy wiki page
- the path can be changed by entering *the relay's full URL* it in the host field
- explain that the URL consists of host colon port slash path

Reply:
I've expanded this section a bit considering your suggestions.

Reply:
"a bit" :D